content/blog/2022/2022-04-20-KaplanMeier.md

```diff
@@ -0,0 +1,73 @@
++++
+author = "Ela Praznik"
+date = "2022-04-15"
+draft = false
+title = "A gentle introduction to the Kaplan-Meier Estimator"
+type = "blog"
+thumbImage = ""
+frontPageImage = ""
```

Comment:
This is required for the preview on the first page. A good ratio is 600x400. You can simply resize/crop an existing image and have it here.